src/Native/Runtime/amd64/AsmOffsetsCpu.h

```diff
@@ -70,23 +70,35 @@ PLAT_ASM_OFFSET(90, REGDISPLAY, Xmm)
 
 #else // !UNIX_AMD64_ABI
 
+#if defined(__APPLE__)
```

Comment:
Nit: Could you please change the define to something like `UNWIND_WORKAROUND` that is defined for `__APPLE__` so that all places related to this are easy to find?